docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,426 @@
+2020-08-24 17:36:40 👶 Starting BABE Authorship worker
+2020-08-24 17:36:45 💤 Idle (0 peers), best: #0 (0xe482…3e12), finalized #0 (0xe482…3e12), ⬇ 0 ⬆ 0
+2
+```
+
+Notice a few things about this output:
+
+- Chain specification is `Rococo Local Testnet`.
+- Validator node name is set to `Alice`.
+- Role is set to `AUTHORITY`. This means is that alice is a validator node.
+- `Local node identity is: 12D3KooWJJzcDNfGi3FXifAhrmZP1baL4heEJAKHZbE6mz7pCmgH` this is **Alice's
+  peer ID** and will be used for setting up other validator nodes.
+- `Prometheus server started at 127.0.0.1:9615`, the `127.0.0.1` is **Alice's IP**.
+- `Idle (0 peers)` will change depending on how many other validators you have running.
+
+To run another validator, you will need the following command in another terminal:
+
+```
+./target/release/polkadot --chain /rococo-#.json --tmp --ws-port 9966 --port 30335 --NODE --bootnodes /ip4/<Alice ID>/tcp/30333/p2p/<Alice peer ID>
+```
+
+> Important note: When you run another validator, change the values for `--ws-port` and `--port`, so
+> that they aren't conflicting with eachother. The Rococo `json` file should be kept the same here.
+
+## Connect to Polkadot JS UI
+
+At this point, you are now ready to connect your local node to the Polkadot JS UI. To do so, head
+over to the [user interface](https://polkadot.js.org/apps/) and change the network to `Local Node`.
+For now, you can change the custom endpoint to one of your validator `--ws-port` values. When this
+the network is connected, the main page should look like this:
+
+![rococo-connected-to-ui](assets/rococo/rococo-ui-connected.png)
+
+## Starting the Parachain with the Relay Chain
+
+In order for the parachain to launch properly, we first need to register it with the Relay Chain. We
+will begin by running this command to generate a WASM file inside your substrate parachain template
+folder:
+
+```
+./target/release/parachain-collator export-genesis-wasm > para-wasm
+```
+
+Next, inside the same terminal, you will need to run the collator node.
+
+```
+./target/release/parachain-collator --tmp --validator --ws-port # --port # --parachain-id 200 -- --chain <Path to>/rococo-#.json --bootnodes /ip4/<Alice IP>/tcp/<Alice port>/p2p/<Alice Peer ID> <Other Relay Chain Node(s)>
+```
+
+> Tip: to check what your validator nodes are doing on the network so far, you can look for:
+> `✍️ Starting parachain attestation session (parent: 0xe482…3e12) with active duty LocalDuty { validation: Relay, index: 0 `
+> in your terminal. In this case, this validator node is working on the Relay Chain.
+
+Note that for `--port` and `--ws-port` you need to set to different values than your validator
+nodes. Also take note that for creating more parachains, you'd need to change the value of the
+`--parachain-id`, and it is not required that the parachain id in this case be `200`, it can be
+whatever id you'd like that to be. The `<Alice port>` should also be replaced with alice's port
```

Comment:
```suggestion
whatever ID you choose. The `<Alice port>` should also be replaced with alice's port
```

The ID needs to be numeric, is there a range that it must be within? Any IDs that won't work?